graftm/create.py

```diff
@@ -198,6 +203,44 @@ def cleanup(self, the_trashcan):
             else:
                 os.remove(every_piece_of_junk)
     
+    def _define_range(self, sequences):
+        '''
+        define_range - define the maximum range within which two hits in a db 
+        search can be linked. This is defined as 1.5X the average length of all
```

Comment:
I wonder if this definition is sufficient. Maybe 1.5X the length of the maximal sequence length? What do you think?